File: SortColors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        a=nums.count(0)
        b=nums.count(1)
        for i in range(0,a):
            nums[i]=0
        for i in range(a,a+b):
            nums[i]=1
        for i in range(a+b,len(nums)):
            nums[i]=2
        return nums
        

File: SortColors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) 
    {
        int size = nums.size();
        
        int low = 0;
        int mid = 0;
        int high = size - 1;
        
        
        
        
        while(mid <= high)
        {
            if(nums[mid] == 0)
            {
                swap(nums[low], nums[mid]);
                mid = mid + 1;
                low = low + 1;
            }
            
            else if(nums[mid] == 1)
            {
                mid = mid + 1;
            }
            
            else if(nums[mid] == 2)
            {
                swap(nums[mid], nums[high]);
                high = high - 1;
            }
        }
        
    }
};